src/object/obj_internal.h

```diff
@@ -235,5 +242,8 @@ obj_dkey2hash(daos_key_t *dkey)
 int obj_ec_codec_init(void);
 void obj_ec_codec_fini(void);
 struct obj_ec_codec *obj_ec_codec_get(daos_oclass_id_t oc_id);
+int daos_encode_full_stripe(daos_obj_id_t oid, daos_sg_list_t *sgl,
```

Comment:
func name mismatch with above "obj_encode_full_stripe".

Reply:
I've made all of the changes requested. Thanks for the constructive feedback.